### PR TITLE
accept twilio messages with special characters

### PR DESCRIPTION
**Pull request by schuyler1d**
This is a workaround from Twilio sending X-Twilio-Body with the full message and AWS/Lambda/express library rejecting the header values as invalid when there are special characters.

Examples of special characters that were marked invalid:
1. `'I\u0019m sorry. I do not know you....`
2. `'Liked \u001cHi, .... ` (from an iMessage feature where the contact can 'like' or 'dislike' a message)

This has been tested on stage.

### CODE REVIEW

**Review by shakalee14 — COMMENTED** [review]
is there a test we can write to accompany this?

**Comment by schuyler1d** [comment]
I added a test (https://github.com/MoveOnOrg/Spoke/pull/1001), but before the election I think we shouldn't merge the test, but we should merge this

**Review by shakalee14 — APPROVED** [review]

**Pull request merged**
